tools/autograd/gen_python_functions.py

```diff
@@ -92,6 +92,7 @@
     'data', 'is_leaf', 'output_nr', '_version', 'requires_grad_', 'retains_grad', 'set_',
     '_fw_primal', 'fake_quantize_per_tensor_affine_cachemask',
     'fake_quantize_per_channel_affine_cachemask',
+    '_reshape_alias',
```

Comment:
Note that this is not necessary as the leading _ means that it won't be dumped in the main namespace.
So if you wanted to write the c++ tests your added in python, you could remove that.
Whichever way you prefer works.